applications/dashboard/modules/class.pagermodule.php

```diff
@@ -64,14 +64,14 @@ class PagerModule extends Gdn_Module {
     public $Offset;
 
     /** @var int The last offset of the current page. (ie. Offset to LastOffset of TotalRecords). */
-    private $_LastOffset;
+    protected $_LastOffset;
```

Comment:
`MorePagerModule` extends `PagerModule`, but was including its own private `$_LastOffset` property. To ensure child classes can share the property, the access specifier was downgraded from private to protected.